src/components/user/uc/registrationPin.uc.js

```diff
@@ -8,7 +8,12 @@ const deleteRegistrationPins = async (pinIds) => {
 	return registrationPinRepo.deleteRegistrationPins(pinIds);
 };
 
+const deleteRegistrationPinsByEmail = async (email) => {
+	return { complete: true, data: registrationPinRepo.deleteRegistrationPinsByEmail(email) };
```

Comment:
deleteRegistrationPinsByEmail it is execute on other position and errors are also handled on other position. Can we say on this position it is complete: true ? Maybe complete at this position is implicit true, but should false. The instances that execute this can say if it is complete.